client/src/App.js

```diff
@@ -1,18 +1,39 @@
-import React, { Component } from 'react';
-import { Route, Link } from 'react-router-dom';
+import React from 'react';
+import { Route } from 'react-router-dom';
+import axios from 'axios';
 import SavedList from './Movies/SavedList';
-// import MovieList from './Movies/MovieList';
-// import Movie from './Movies/Movie';
+import MovieList from './Movies/MovieList';
 import MovieCard from './Movies/MovieCard';
+import {withRouter} from 'react-router-dom';
```

Comment:
withRouter is a higher order component that we can use to gain access to history, location, and match in components that are not directly rendered by Route. You could technically export your app inside of withRouter() and pass down the id on match.params.id into your MovieCard component